cmd/charm/charmcmd/show.go

```diff
@@ -250,7 +250,9 @@ func newShowData(out io.Writer, metadata map[string]interface{}) showData {
 	if val, ok := metadata["charm-metadata"]; ok {
 		charmMetadata := val.(map[string]interface{})
 		sd.summary = charmMetadata["Summary"].(string)
-		sd.tags = toStringArray(charmMetadata["Tags"].([]interface{}))
+		if val, ok := charmMetadata["Tags"]; ok {
```

Comment:
how about?
```
if tags, ok := charmMetadata["Tags"].([]interface{}); ok {
	sd.tags = toStringArray(tags)
}
```
That copes with a null Tags value too, although I hope you'd never receive one.